src/Tribe/Admin/Move_Tickets.php

```diff
@@ -672,8 +672,8 @@ public function move_tickets( array $ticket_ids, $tgt_ticket_type_id, $src_event
 			if ( 'Tribe__Tickets__RSVP' === $ticket['provider'] ) {
 				$src_stock ++;
 				$tgt_stock --;
-				update_post_meta( $src_ticket_type_id, '_stock', $src_stock );
-				update_post_meta( $tgt_ticket_type_id, '_stock', $tgt_stock );
```

Comment:
So we dont change the ticket stock too? Just the Event global Stock?

Reply:
@bordoni The '_stock' meta, i think doesn;t have any value in an ticket-able post type. They use diffenent meta for that. I think its `tribe( 'tickets.handler' )->key_capacity` and `Global_Stock::GLOBAL_STOCK_LEVEL`.

Reply:
@bordoni maybe we need to update both meta